Qualifyze/design-system
# VE-512 Add new tag design

# What❓

Updated tags to new design

# Why 💁‍♀️

We need new tag design

# How to test ✅

Use to component or check out preview at http://qualifyze-app-preview-1917.s3-website.eu-central-1.amazonaws.com/audit/recDRv5wqprV6Wd5V

# What was tested / affected 📝

Tag design
